packages/graphics/gammastep/package.mk

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-present ROCKNIX (https://github.com/ROCKNIX)
+
+PKG_NAME="gammastep"
+PKG_VERSION="master"
```

Comment:
why not using v2.0.9 tag ?
doesn't seems like master head is gonna be updated soon but...

Reply:
I initially did have them as seperate; but my git foo is weak and was trying to collapse commits into a single PR as per previous requests. 

I didn't set a tag version because it's a development tool at this point which requires specific pulling into build options, so until it's confirmed useful and integrated properly seemed reasonable to pull from whatever latest version of upstream.